### PR TITLE
create hero Section

### PR DESCRIPTION
**Pull request by daudb6**
hello, i create hero section. please review it

### CODE REVIEW

File: src/Component/Navbar/Navbar.css [hunk 1]
```diff
@@ -2,7 +2,7 @@ h3 {
   font-size: 6rem;
 }
 .navbar {
-  background-color: aqua;
+  background-color: rgb(98, 255, 0);
```
Comment (muneeb3123) [inline src/Component/Navbar/Navbar.css]: please do not change color its not looking good

**Review by muneeb3123 — CHANGES_REQUESTED** [review]
you do good work but i am not satified please change it

File: src/Component/Navbar/Navbar.css (hunk 1 above)
Comment (daudb6) [reply src/Component/Navbar/Navbar.css]: > please do not change color its not looking good

ok done

**Review by muneeb3123 — APPROVED** [review]
done

**Pull request merged**